packages/nimbus/src/components/image/image.mdx

```diff
@@ -12,208 +12,77 @@ tags:
 
-| Value     | Description                                                                                                                          |
-| :-------- | :----------------------------------------------------------------------------------------------------------------------------------- |
-| `cover`   | The default. The image keeps its aspect ratio and fills the given dimension. The image will be _clipped_ to fit.                     |
-| `fill`    | The image is resized to fill the given dimension. If necessary, the image will be _stretched_ or _squished_ to fit.                  |
-| `contain` | The image _keeps its aspect ratio_, but is resized to fit within the given dimension,<br/> potentially leaving some space uncovered. |
-| `none`    | The image is not resized.                                                                                                            |
+### Best practices
 
-Here is a demo of the different fit values, the red outline shows the allocated space:
+- **Relevance:** Ensure the image directly supports or enhances the content it is paired with; avoid purely decorative or confusing visuals.
+- **Quality and Format:** Use high-quality, appropriately sized images.
+  - **JPEG/WebP:** For photographs and complex images.
+  - **PNG:** For images requiring transparency or simple graphics.
+  - **SVG:** For logos, icons, and illustrations that need crisp scaling.
+- **Responsiveness:** Implement responsive design principles so images scale correctly and maintain aspect ratios across various screen sizes without distorting or breaking the layout.
+- **Performance optimization:** Compress images to minimize file size and ensure fast loading times. Implement lazy loading for images that appear below the fold.
+- **Visual hierarchy:** Images should contribute to, but not dominate, the visual hierarchy unless they are the primary content (e.g., product hero image).
+- **Maintain brand style:** Ensure all illustrations and graphics adhere to the established brand style and color palette.
 
-```jsx-live
-const App = () => (
-  <Stack direction="row" spacing="200">
-   <Image
-      boxSize="128px"
-      fit="cover"
-      src="https://i.imgur.com/fqpnfaN.png"
-      alt="Covered image"
-      outline="1px solid red"
-    />
-    <Image
-      boxSize="128px"
-      fit="fill"
-      src="https://i.imgur.com/fqpnfaN.png"
-      alt="Filled image"
-      outline="1px solid red"
-    />
-    <Image
-      boxSize="128px"
-      fit="contain"
-      src="https://i.imgur.com/fqpnfaN.png"
-      alt="Contained image"
-      outline="1px solid red"
-    />    
-    <Image
-      boxSize="128px"
-      fit="none"
-      src="https://i.imgur.com/fqpnfaN.png"
-      alt="Filled image"
-      outline="1px solid red"
-    />
-  </Stack>
-);
-```
+## Specs
 
-#### Align
+<PropsTable id="Image" />
```

Comment:
Should this go ahead into a `.dev.mdx` file?

Reply:
It should, definitely. But I think it should be tackled in the ticket @valoriecarli mentioned, or as its own unit of work for the engineering docs for this component